src/data/content/design-system/componenti/header.yaml

```diff
@@ -126,7 +153,17 @@ tabs:
             title: Comportamento
             headingLevel: 2
             text: |
-              *In stesura*
+              Tutte le varianti di header sono **responsive**, quindi si adattano alla viewport di qualsiasi dispositivo.
+
+              ### Interazioni 
+
+              Su dispositivi mobile o touch, il menu di navigazione non è direttamente visibile in pagina ma si apre tramite click/tap sull’icona menu, con un’animazione di ingresso da sinistra verso destra.
+
+              I link del menu di navigazione che prevedono un sottomenu espandibile ereditano le interazioni del componente [dropdown](#).
+
+              L’accesso alla barra di ricerca può avvenire in due modi: 
+              - tramite link alla pagina dedicata;
+              tramite l’apertura di una finestra (_overlay_) rimanendo nella stessa pagina .
```

Comment:
C'è un problema di impaginazione in questo elenco. La seconda riga manca del trattino `-` iniziale.